lib/provider/vpc_data_types.go

```diff
@@ -43,9 +45,15 @@ type Profile struct {
 
 // VolumeAttachment ...
 type VolumeAttachment struct {
-	Href string `json:"href,omitempty"`
-	ID   string `json:"id,omitempty"`
-	Name string `json:"name,omitempty"`
-
-	Volume *Volume `json:"volume,omitempty"`
+	Href       string `json:"href,omitempty"`
+	ID         string `json:"id,omitempty"`
+	Name       string `json:"name,omitempty"`
+	InstanceID string `json:"instance_id,omitempty"`
+
+	Volume    *Volume    `json:"volume,omitempty"`
+	Status    string     `json:"status,omitempty"` //attaching, attached, detaching
```

Comment:
Looks duplicate of `Status` as parent struct also has the same

Reply:
Remove status from Parent